File: Wrap-up/python/armstrong_number_between_one_and_a_thousand.py
import math

for count in range (1, 1001):
    storeOne = count
    storetwo = count
    product = 0
    sum = 0
    counter = 0
    
    while(storeOne != 0):
        digits = storeOne % 10
        
        counter += 1
        
        storeOne = storeOne // 10
    
    
    while(storetwo != 0):
        digits = storetwo % 10;
        
        product = math.pow(digits, counter);
        
        sum += product;
        
        storetwo = storetwo // 10;
    
    
    if(count == sum):
        print(count, " ");
    

File: Wrap-up/Javascript/ArmstrongNumberBetweenOneAndAThousand.js
for(let count = 1; count <= 1000; count ++){
    let storeOne = count;
    let storetwo = count;
    let product = 0;
    let sum = 0;
    let counter = 0;
    
    while(storeOne != 0){
        let digits = storeOne % 10;
        
        counter++;
        
        storeOne = Math.floor(storeOne / 10);
    }
    
    while(storetwo != 0){
        let digits = storetwo % 10;
        
        product = Math.pow(digits, counter);
        
        sum += product;
        
        storetwo = Math.floor(storetwo / 10);
    }
    
    if(count == sum){
        console.log(count + " ");
    }
}

